uefi/CHANGELOG.md

```diff
@@ -60,7 +64,7 @@
 - `Mode` is now `Copy` and `Clone`.
 - Added `TryFrom<&[u8]>` for `Time`.
 
-## Changed
+## Changed (**BREAKING**)
```

Comment:
Can drop the ` (**BREAKING**)` from these section headers

Reply:
Yikes. I wonder why it still exists. I removed all of them. Probably something during the rebase failed. Thanks!